### DISCUSSION TITLE
Lesson 9 Chainlink-brownie

### DISCUSSION THREAD

**Discussion by Darshposhiya29** [Cyfrin/foundry-full-course-cu#238] [Q&A] [answered]
forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit

I have run this command it is installed but is not showing in lib folder

when I manually go by doing cd & ls it is there in the lib folder but it is not showing the explorer

**Comment by Shiva-Sai-ssb** [comment, marked as answer]
@Darshposhiya29 , Visual Studio Code can sometimes be buggy. I suggest closing and then reopening the application to see if that resolves the issue.
